website/docs/docs/dbt-versions/release-notes/04-Sept-2023/product-docs-summer-rn.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Summer 2023 product documentation updates"
+id: "product-docs-summer"
+description: "Summer 2023: The Product docs team merged 256 PRs, made various updates to dbt Cloud and Core, such as adding What's New, writing Semantic Layer beta docs, releasing dbt 1.6 docs and more!"
+sidebar_label: "Update: Product docs changes"
+tags: [July-2023, Aug-2023, product-docs]
+date: 2023-09-07
+sidebar_position: 09
+---
+
+Hello from dbt's Product Documentation team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! We'd like to recognize all of the docs and support from our partner team, Developer Experience: @jasnonaz @gwenwindflower @dbeatty10 @dataders @joellabes @Jstein77 @dave-connors-3! 
+
+What a busy summer! We merged 256 PRs between July 1st and August 31. We’d like to thank the 22 community members who contributed for the 1st time to docs.getdbt.com :pray: Here's what's new to [docs.getdbt.com](http://docs.getdbt.com/):
+
+* Changes based on YOUR feedback (you can reach us most directly by opening a pull request or issue in [our repo](https://github.com/dbt-labs/docs.getdbt.com) or reaching out in the dbt community Slack channel #dbt-product-docs): 
```

Comment:
adding link to slack channel. also suggesting to add the 'feedback callout' at the end of this

```suggestion
* Changes based on YOUR feedback (you can reach us most directly by opening a pull request or issue in [our repo](https://github.com/dbt-labs/docs.getdbt.com) or reaching out in the dbt community Slack channel [#dbt-product-docs](https://getdbt.slack.com/archives/C0441GSRU04)): 
```